src/shx.js

```diff
@@ -16,6 +18,25 @@ export const shx = (argv) => {
     return EXIT_CODES.SHX_ERROR;
   }
 
+  // Load ShellJS plugins
+  const CONFIG_PATH = path.join(process.cwd(), CONFIG_FILE);
+  if (fs.existsSync(CONFIG_PATH)) {
```

Comment:
`fs.exists` is [deprecated](https://nodejs.org/api/fs.html#fs_fs_exists_path_callback), should wrap `fs.accessSync()` in a try catch instead or use the async version if possible.